includes/internal/student-progress/quiz-progress/repositories/class-comments-based-quiz-progress-repository.php

```diff
@@ -78,6 +79,8 @@ public function get( int $quiz_id, int $user_id ): ?Quiz_Progress {
 			Quiz_Progress::STATUS_GRADED,
 			Quiz_Progress::STATUS_PASSED,
 			Quiz_Progress::STATUS_UNGRADED,
+			// Need to support legacy status from lessons as some code still relies on it.
+			Lesson_Progress::STATUS_COMPLETE,
```

Comment:
@m1r0 I had to add this status here to make substitution of lesson progress with quiz progress working.
Won't be needed after switching to separate models with consistent behavior.